Carrot-Assistant/app.py

```diff
@@ -34,16 +35,21 @@
     allow_headers=["*"],
 )
 
-# the LLMModel class could be more powerful if we remove the option of using the GPTs. Then it can be an Enum of dicts, and the dicts can be unpacked into the arguments for the hf download
+
+# the LLMModel class could be more powerful if we remove the option of using the GPTs. /
+# Then it can be an Enum of dicts, and the dicts can be unpacked into the arguments for the hf download
+
+
 class LLMModel(str, Enum):
     """
     This enum holds the names of the different models the assistant can use
     """
-    GPT_3_5_TURBO = "gpt-3.5-turbo-0125",
-    GPT_4 = "gpt-4",
-    LLAMA_2_7B = "llama-2-7B-chat",
-    LLAMA_3_8B = "llama-3-8B",
-    LLAMA_3_70B = "llama-3-70B",
+
+    GPT_3_5_TURBO = ("gpt-3.5-turbo-0125",)
```

Comment:
Why are these all in tuples now?

Reply:
Thats was another combination I was trying to implement. It's a mistake and I'll change it.